### PR TITLE
Corrections to Liber Enoch

### PR DESCRIPTION
**Pull request by sonofmun**
Including added the Swete version of the Greek

### CODE REVIEW

**Comment by capitains-hook** [comment]
## Global

| Changed            |   Status |
|:-------------------|---------:|
| `metadata_passing` |      +13 |
| `metadata_total`   |      +13 |
| `nodes_count`      |    +8055 |
| `texts_passing`    |      +15 |
| `texts_total`      |      +15 |

## Words

| Changed   |   Status |
|:----------|---------:|
| `deu`     |    -1102 |
| `grc`     |  +109949 |
| `lat`     |     +200 |

## Units

| Changed                                               | Status   |
|:------------------------------------------------------|:---------|
| `./data/tlg0527/tlg021/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg021/tlg0527.tlg021.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg023/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg023/tlg0527.tlg023.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg025/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg025/tlg0527.tlg025.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg026/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg026/tlg0527.tlg026.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg027/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg027/tlg0527.tlg027.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg028/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg028/tlg0527.tlg028.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg029/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg029/tlg0527.tlg029.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg031/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg031/tlg0527.tlg031.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg032/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg032/tlg0527.tlg032.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg033/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg033/tlg0527.tlg033.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg035/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg035/tlg0527.tlg035.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg037/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg037/tlg0527.tlg037.1st1K-grc1.xml` | New      |
| `./data/tlg0527/tlg038/__cts__.xml`                   | New      |
| `./data/tlg0527/tlg038/tlg0527.tlg038.1st1K-grc1.xml` | New      |
| `./data/tlg1463/tlg001/tlg1463.tlg001.1st1K-grc1.xml` | New      |
| `./data/tlg1463/tlg001/tlg1463.tlg001.1st1K-lat1.xml` | New      |
| `./data/tlg0527/tlg021/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg021/tlg0527.tlg021.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg023/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg023/tlg0527.tlg023.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg025/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg025/tlg0527.tlg025.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg026/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg026/tlg0527.tlg026.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg027/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg027/tlg0527.tlg027.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg028/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg028/tlg0527.tlg028.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg029/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg029/tlg0527.tlg029.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg031/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg031/tlg0527.tlg031.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg032/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg032/tlg0527.tlg032.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg033/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg033/tlg0527.tlg033.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg035/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg035/tlg0527.tlg035.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg037/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg037/tlg0527.tlg037.1st1K-grc1.xml` | Passing  |
| `./data/tlg0527/tlg038/__cts__.xml`                   | Passing  |
| `./data/tlg0527/tlg038/tlg0527.tlg038.1st1K-grc1.xml` | Passing  |
| `./data/tlg1463/tlg001/tlg1463.tlg001.1st1K-grc1.xml` | Passing  |
| `./data/tlg1463/tlg001/tlg1463.tlg001.1st1K-lat1.xml` | Passing  |

*[Hook UI build recap](http://ci.perseids.org/repo/OpenGreekAndLatin/First1KGreek/3025)*

File: data/tlg1463/tlg001/tlg1463.tlg001.opp-grc1.xml [hunk 1]
```diff
@@ -72,17 +72,18 @@
    </editor>
 	  <editor>
 	<persName>
-	<name xml:lang="deu">Ludwig Radermacher</name>
```
Comment (annettegessner) [inline data/tlg1463/tlg001/tlg1463.tlg001.opp-grc1.xml]: Why did you delete these two lang-tags here and in line 81? Aren't they correct?

File: data/tlg1463/tlg001/tlg1463.tlg001.1st1K-lat1.xml [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+<pubPlace>Germany</pubPlace>
+</publicationStmt>
+<sourceDesc>
+<listBibl>
+ <biblStruct>
+  <monogr>
+   <title xml:lang="deu">Das Buch Henoch</title>
+   <editor>
+	<persName>
+	 <name xml:lang="lat">Johannes Flemming</name>
```
Comment (annettegessner) [inline data/tlg1463/tlg001/tlg1463.tlg001.1st1K-lat1.xml]: Is this the correct lang-tag?

File: data/tlg1463/tlg001/tlg1463.tlg001.opp-grc1.xml (hunk 1 above)
Comment (annettegessner) [reply data/tlg1463/tlg001/tlg1463.tlg001.opp-grc1.xml]: I guess, we don't need them.

File: data/tlg1463/tlg001/tlg1463.tlg001.1st1K-lat1.xml (hunk 2 above)
Comment (sonofmun) [reply data/tlg1463/tlg001/tlg1463.tlg001.1st1K-lat1.xml]: It was already in the file. And if you look near the top of the file, his name is given as "Johann Flemming". Since the Johann was changed to Johannes, I assumed the "lat" language tag was correct. If you think that is not the case, then I will remove it.

Comment (annettegessner) [reply data/tlg1463/tlg001/tlg1463.tlg001.1st1K-lat1.xml]: Interesting. I am not sure about the Latinization from Johann to Johannes, but let's leave it like this.
